packages/arakoon/arakoon.1.8.6/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "1.2"
+name: "arakoon"
+version: "1.8.6"
+maintainer: "Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+authors: ["Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+          "Jan Doms <jan.doms@openvstorage.com>"]
+homepage: "https://github.com/openvstorage/arakoon"
+bug-reports: "https://github.com/openvstorage/arakoon"
+license: "Apache 2.0"
+dev-repo: "https://github.com/openvstorage/arakoon.git"
+build: [
+  [make]
+  
+]
+
+install: [make "install"   "OCAML_LIBDIR=%{arakoon:lib}%" "PREFIX=%{prefix}%"]
+
+remove:  [
+  [make "uninstall_client" "OCAML_LIBDIR=%{arakoon:lib}%" "PREFIX=%{prefix}%"]
+  ["rm" "-rf" "%{bin}%/arakoon"]
+]
+
+depends: [
+  "ocamlfind"      {build}
+  "bisect"         { >= "1.3"}
+  "camltc"         { >= "0.9.2" }
+  "camlbz2"        { >= "0.6.0" }
+  "conf-libev"     { >= "4-11"}
+  "cstruct"        { >= "1.6.0"}
+  "ctypes"         { >= "0.4.0" }
+  "ctypes-foreign" { >= "0.4.0"}
+  "lwt"            { = "2.4.8" }  # 2.4.9 had an incompatible API change
+  "ocplib-endian"  { >= "0.8" }
+  "ssl"            { >= "0.5.0" }
+  "snappy"         { >= "0.1.0" }
+  "quickcheck"     { >= "1.0.2" }
```

Comment:
no. Arakoon just generates 1 binary. You can run that binary 
- as a node of an arakoon cluster
- for tests
- as an ad-hoc cli client
- as an administrative tool
- ...